src/http/http_rpc_context.h

```diff
@@ -318,10 +335,6 @@ namespace http
     {
       ctx.set_method(remaining_path);
     }
-    else
-    {
-      ctx.set_method(path);
-    }
```

Comment:
Why is this removed? This is to detect all other paths that don't have an `actor` i.e. this enables the removal of `app` actor. You should add the new `acme_challenge` actor here https://github.com/microsoft/CCF/blob/main/include/ccf/actors.h#L20

Reply:
Unless I am missing something, it seemed to me this branch of the `else` is a no-op, hence why I removed it.

`path` is defined at the top of the function as:
```
const auto path = ctx.get_method();
```
and neither `path` nor ctx's method get modified until this point, so the `else` block was simply re-assigning the same path value back into ctx's method.

There are other early returns in this function that don't do this assignement, and they work fine. I actually found the disparity a little confusing when I first looked at that code.

Reply:
Ah, yes `path` was never reassigned (and I see that `actors.h` is also updated). Looks good to me.